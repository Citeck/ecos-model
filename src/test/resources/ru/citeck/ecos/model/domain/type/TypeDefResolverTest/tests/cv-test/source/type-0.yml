---
id: type-0
sourceId: emodel/type-0-src

createVariantsForChildTypes: true
createVariants:
  - id: type-0-cv
    formRef: uiserv/form@abc
