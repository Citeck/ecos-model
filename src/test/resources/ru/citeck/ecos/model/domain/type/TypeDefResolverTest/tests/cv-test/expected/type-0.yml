---
id: type-0
name: type-0
sourceId: emodel/type-0-src
metaRecord: emodel/type-0-src@
createVariantsForChildTypes: true
inheritCreateVariants: false
createVariants:
  - id: type-0-cv
    formRef: uiserv/form@abc
    name: {"en":"type-0"}
    sourceId: emodel/type-0-src
    typeRef: emodel/type@type-0
  - id: type-1__type-1-cv
    formRef: uiserv/form@abcdef
    name: {"en":"type-1"}
    sourceId: emodel/type-0-src
    typeRef: emodel/type@type-1
